src/views/OnboardingOrganization.js

```diff
@@ -195,38 +225,61 @@ const OrganizationStepEmail = ({ values, onDisabledChange, onChange }) => {
           </Box>
         </Box>
       </Box>
-    </>
+    </Box>
   );
 };
 
 const OrganizationStepAvatar = ({ values, onDisabledChange, onChange }) => {
+  const useStyles = makeStyles((theme) => ({
+    organizationStepAvatarContainer: {
+      position: 'relative',
+      zIndex: theme.zIndex.layer1,
+    },
+  }));
+  const classes = useStyles();
+
   const handleUpload = (avatarUrl) => {
     onChange({
       avatarUrl,
     });
   };
 
+  const ORGANIZATION_RING_SIZES = {
+    tiny: '77%',
+    small: '108%',
+    medium: '108%',
+    large: '108%',
+  };
+
   return (
-    <>
-      <Typography align="center" gutterBottom variant="h2">
+    <Box className={classes.organizationStepAvatarContainer}>
+      <Typography align="center" gutterBottom variant="h6">
         {translate('OnboardingOrganization.headingAvatar')}
       </Typography>
-      <Typography>{translate('OnboardingOrganization.bodyAvatar')}</Typography>
-      <Box mt={4}>
+      <Box mb={4} mt={4}>
         <AvatarUploader
+          shouldHaveIndicator
```

Comment:
Does this mean it is set to true?

Reply:
yes exactly, any atribute without value by default is 'true'